docs/5.x/installation.md

```diff
@@ -216,15 +207,43 @@ Tue Apr 10 14:01:09 UTC	Executing install phase "/" finished in 5 minutes
 
 ```
 
-The following CLI flags are useful to manage the install operation:
+To abort an installation, press Ctrl+C two times in a row in the terminal.
+If the operation is aborted, the partial cluster state will be automatically removed.
 
+Aborting an agent (and not the installer process), will only prevent this node from joining.
+
+!!! warning "Aborting an agent":
+    Aborting an agent might result in installation failure.
+    If the agent was aborted to correct a configuration error, it can be restarted once the error
+    has been fixed.
+
+
+Installer processes (that includes the main installer process and all agent processes that join additional nodes to the cluster)
+execute inside a systemd service so the operation will continue in background even if the terminal session has timed out.
+
+In order to reconnect to the installer, issue `./gravity resume`.
+
+!!! warning "Installer state directory":
+    It is important to run install commands from the directory with the original gravity binary.
+    This directory contains the temporary operation state required for all commands to work properly.
+
+
+#### Under the hood
+
+Each node runs a systemd service (and may even run two). The main installer service executes
+from a service called `gravity-installer` (agent nodes have a corresponding `gravity-agent`). The services are configured
+from the temporary state sub-directory called `.gravity` in the directory where the installer has been extracted.
+
+To manually remove installation state:
+
+```bash
+# Stop the installer service (or gravity-agent - depending on the node):
+root$ systemctl stop gravity-installer.service
```

Comment:
TODO: maybe this should be incorporated as part of `./gravity leave`?

Reply:
It is already part of `gravity leave` (more specifically, it is part of `gravity system uninstall` which `gravity leave` implicitly executes).